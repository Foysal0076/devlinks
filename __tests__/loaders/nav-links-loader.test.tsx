import { render } from '@testing-library/react'
import NavlinksLoader from '@/components/loaders/nav-links-loader'

test('renders NavlinksLoader correctly', () => {
  const { asFragment } = render(<NavlinksLoader />)
  expect(asFragment()).toMatchSnapshot()
})
