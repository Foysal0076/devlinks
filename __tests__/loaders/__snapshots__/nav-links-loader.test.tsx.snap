// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders NavlinksLoader correctly 1`] = `
<DocumentFragment>
  <div
    class="flex items-center gap-2"
  >
    <div
      class="bg-loader h-10 w-[3.125rem] animate-pulse rounded-md md:h-11 md:w-[6rem]"
    />
    <div
      class="bg-loader h-10 w-[3.125rem] animate-pulse rounded-md md:h-11 md:w-[6rem]"
    />
    <div
      class="bg-loader h-10 w-[3.125rem] animate-pulse rounded-md md:h-11 md:w-[6rem]"
    />
  </div>
</DocumentFragment>
`;
